sdk-lib/src/test/java/cash/z/ecc/android/sdk/model/BlockHeightTest.kt

```diff
@@ -68,4 +68,32 @@ class BlockHeightTest {
             ZcashNetwork.Mainnet.saplingActivationHeight + -1L
         }
     }
+
+    @Test
+    fun subtraction_of_block_height_succeeds() {
+        val one = BlockHeight.new(
+            ZcashNetwork.Mainnet,
+            ZcashNetwork.Mainnet.saplingActivationHeight.value +
+                ZcashNetwork.Mainnet.saplingActivationHeight.value
+        )
+        val two = BlockHeight.new(ZcashNetwork.Mainnet, ZcashNetwork.Mainnet.saplingActivationHeight.value)
+
+        assertEquals(ZcashNetwork.Mainnet.saplingActivationHeight.value, (one - two).value)
+    }
+
+    @Test
+    fun subtraction_of_long_succeeds() {
+        assertEquals(
+            ZcashNetwork.Mainnet.saplingActivationHeight.value,
+            (BlockHeight(419_323L) - 123L).value
+        )
+    }
+
+    @Test
+    fun subtraction_of_int_succeeds() {
+        assertEquals(
+            ZcashNetwork.Mainnet.saplingActivationHeight.value,
+            (BlockHeight(419_323) - 123).value
+        )
+    }
```

Comment:
Check that subtracting a negative value fails.

Reply:
That is already covered in lines 58-70 with the older tests `subtraction_of_int_fails` and `subtraction_of_long_fails` tests.